### ISSUE TITLE
Server url with CNAME record

### ISSUE THREAD

**Issue by Dextroz** [jitsi/docker-jitsi-meet#432]
I added a CNAME record to the domain I have for my instance. 
When joining a meeting a lot of issues started occurring. Video and voice doesn’t work and there is no prompt to authenticate the user to start the meeting.

**Comment by saghul** [comment]
How is the CNAME record related to any of that? Can you share some client JS console logs?

**Comment by Dextroz** [comment]
Well I thought it may be due to it because when I use the original domain to access Jitsi, functionality is perfect.

**Comment by Dextroz** [comment]
I don’t have the means currently to send the logs at this time. However, I am seeing errors in the console.
I will send them tomorrow. Thanks for the quick reply! 👍🏻

**Comment by Dextroz** [comment]
![image](https://user-images.githubusercontent.com/25749668/79038679-16db2d00-7bd3-11ea-8899-f1bfe183830e.png)
@saghul Is this helpful at all?

**Comment by Dextroz** [comment]
@saghul Is this something which cannot be resolved? 

I was looking at the meet.conf where the static assets are assigned the `Access-Control-Allow-Origin *` header. Maybe something along these lines needs to be added to the `/http-bind` section of the config to resolve my issue?

I see the repo is exploding with issues, most likely due to people looking for alternatives to Zoom etc.

**Comment by Dextroz** [comment]
Tried it, didnt work. Well was worth a try 🤷

**Comment by Dextroz** [comment]
So I managed to get it working @saghul 

First I had the error as shown above. To fix this I added the following to `ssl.conf`:
`add_header Access-Control-Allow-Origin "https://jitsi.myexample.com";`

Then I had another error saying CORS policy failed preflight checks as the header `content-type` wasn't allowed. To fix this, I added the following to `ssl.conf` again:
`add_header Access-Control-Allow-Headers "content-type";`

Restarted the web container and it is now working correctly via my CNAME record.
